examples/chat/client/cpp/chat.cc

```diff
@@ -82,14 +81,14 @@ void ListenLoop(Chat::Stub* stub, const RoomId& room_id, const std::string& user
     }
   }
   done->set(true);
-  std::cout << "\n\nRoom " << room_id << " closed.\n\n";
+  std::cout << "\n\nRoom closed.\n\n";
```

Comment:
I think there is not much value in the id actually, since each client only deals with one room, which is specified on the command line. I think the point of the logs is to help with debugging.

cout vs LOG: I don't remember, but looking it at it seems that cout is used for the actual user-facing output of the chat app (messages etc.), while LOG for additional logging.